File: 프로그래머스/1/12931. 자릿수 더하기/자릿수 더하기.cpp
#include <iostream>
#include <string>
#include <vector>

using namespace std;
int solution(int n)
{
    int answer = 0;

    vector<char> element;
    string numStr;
    
    numStr = to_string(n);
    for(int i=0; i<numStr.length(); ++i)
    {
        element.push_back(numStr[i]);
    }
    
    for(int i=0; i<element.size(); ++i)
    {
        answer += element[i] - '0';
    }
    
    // [실행] 버튼을 누르면 출력 값을 볼 수 있습니다.
    cout << "Hello Cpp" << endl;

    return answer;
}

File: 프로그래머스/1/12931. 자릿수 더하기/자릿수 더하기.cs
using System;

public class Solution {
    public int solution(int n) {
        int answer = 0;
        
        string str = n.ToString();
        int length = str.Length;
        
        for(int i = 0; i < length; ++i)
            answer += str[i] - '0';
        
        return answer;
    }
}